src/main/java/org/andersen/homework/model/dto/user/UserSaveDto.java

```diff
@@ -0,0 +1,4 @@
+package org.andersen.homework.model.dto.user;
+
+public class UserSaveDto {
```

Comment:
Why do we need empty class?

Reply:
It's empty because `User` has only `id` field, that is not required for user creation. To save new `User` you need just empty dto. In the future `User` entity may become populated with some new fields and it will be easy to populate this dto also without any changes in services and controllers layers

Reply:
Why user and admin are pure empty objects? 
From coding perspective I get the suggestion, but from business couldn't imagine that such situation happened...
At least each user has username, role